plugins/ext_test/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Having a 2nd license might prove problematic within the same Git repo might prove problematic for some large enterprise organizations.  However, since they are both MIT it might be OK.